docs/Migrating.md

```diff
@@ -24,6 +24,12 @@ The versions can be found in
 * `BrainParameters` and `SpaceType` have been removed from the public API
 * `BehaviorParameters` have been removed from the public API.
 * `DecisionRequester` has been made internal (you can still use the DecisionRequesterComponent from the inspector). `RepeatAction` was renamed `TakeActionsBetweenDecisions` for clarity.
+* The following methods in the `Agent` class were renamed:
```

Comment:
Should we clarify that they've been deprecated and will be removed in a later version?